nservicebus/rabbitmq/configuration-api.md

```diff
@@ -14,197 +14,242 @@ To use RabbitMQ as the underlying transport:
 #### UserName
 
-The username when connecting. 
+The user name to use to connect to the broker.
 
-Defaults: `guest`
+Default: `guest`
 
 
 #### Password
 
-The password when connecting. 
+The password to use to connect to the broker.
 
-Defaults: `guest`
+Default: `guest`
 
 
 #### RequestedHeartbeat
 
 The interval for the heartbeats between the client and the server.
 
-Defaults: `5` seconds
+Default: `5` seconds
 
 
 #### DequeueTimeout
 
 The time period allowed for the dequeue strategy to dequeue a message.
 
-Defaults: `1` second
+Default: `1` second
 
 Versions: 3 and below.
 
 
 #### PrefetchCount
 
-The number of messages to [prefetch](http://www.rabbitmq.com/consumer-prefetch.html) when consuming messages from the broker. 
+The number of messages to [prefetch](http://www.rabbitmq.com/consumer-prefetch.html) when consuming messages from the broker.
 
-Defaults: The number of configured threads for the transport(as of v2.1)
+Default: The number of configured threads for the transport(as of v2.1)
 
-Versions: 3 and below. In Versions 4 and above `EndpointConfiguration.LimitMessageProcessingConcurrencyTo` can be used instead. See [Throughput Throttling](/samples/throttling/) sample.
+Versions: 3 and below.
 
 
 #### UsePublisherConfirms
 
 Controls if [publisher confirms](https://www.rabbitmq.com/confirms.html) should be used. 
 
-Defaults: `true`
+Default: `true`
 
 
 #### MaxWaitTimeForConfirms
 
-How long the client should wait for publisher confirms if enabled. 
+How long the client should wait for publisher confirms, if enabled.
+
+Default: `30` seconds.
 
-Defaults: `30` seconds.
+Versions: 3 and below.
 
 
 #### RetryDelay
 
-The time to wait before trying to reconnect to the broker if connection is lost.
+The time to wait before trying to reconnect to the broker if the connection is lost.
 
-Defaults: `10` seconds
+Default: `10` seconds
 
 
 #### UseTls
 
-Indicates if the connection should be done using TLS. 
+Indicates if the connection to the broker should be secured with [TLS](/nservicebus/rabbitmq/configuration-api.md#specifying-the-connection-string-transport-layer-security-support).
 
-Default: If `Port` is not specified and `UseTls` is enabled the port will default to `5671`.
+Default: `false`
 
 Versions: 4 and above
 
 
 #### CertPath
 
-The certificated path that should be used when using TLS.
+The file path to the client authentication certificate when using [TLS](/nservicebus/rabbitmq/configuration-api.md#specifying-the-connection-string-transport-layer-security-support)
 
 Versions: 4 and above
 
 
 #### CertPassphrase
 
-The certificate password.
+The password for the client authentication certificate specified in `CertPath`
 
 Versions: 4 and above
 
 
-## Specifying the connection string
-
-To use a custom name for the connection string use:
-
-snippet:rabbitmq-config-connectionstringname
+NOTE: For debugging purposes, it can be helpful to increase the `RequestedHeartbeat` and `DequeueTimeout` settings like this:
```

Comment:
How about:

NOTE: For debugging purposes, it can be helpful to increase the `RequestedHeartbeat` and `DequeueTimeout` as shown below:

Also any reasons on why it can be helpful to increase those values?